libs/designsystem/src/lib/kirby.module.ts

```diff
@@ -45,9 +45,26 @@ import {
   ProgressCircleRingComponent,
 } from '@kirbydesign/designsystem/progress-circle';
 import { RouterOutletModule } from '@kirbydesign/designsystem/router-outlet';
-import { AppModule } from './components/app/app.module';
-import { ButtonComponent } from './components/button/button.component';
-import { CalendarComponent } from './components/calendar/calendar.component';
+import { ButtonComponent } from '@kirbydesign/designsystem/button';
+import { CalendarComponent } from '@kirbydesign/designsystem/calendar';
+import {
+  ActionSheetComponent,
+  ActionSheetHelper,
+  AlertComponent,
+  AlertHelper,
+  ModalController,
+  ModalFooterComponent,
+  ModalHelper,
+} from '@kirbydesign/designsystem/modal';
+import {
+  ModalCompactWrapperComponent,
+  ModalWrapperComponent,
+} from '@kirbydesign/designsystem/modal';
```

Comment:
Can't the imports for `ModalCompactWrapperComponent` and `ModalWrapperComponent` be moved to the import on line 58?